io/plot_trial_raster.m

```diff
@@ -0,0 +1,27 @@
+function [cellOrder] = plot_trial_raster(ic_traces,binned_frames,trial_num)
```

Comment:
`frame_indices` would be more general than `binned_frames`, since we may at some point run this function without time binning. (The requirement is that the frame indices match the length of traces.)

Also, add a brief comment about the function? The expected format of `ic_traces` (i.e. `[num_frames x num_ICs]`), etc. would be helpful.